plaidml2/exec/ffi.cc

```diff
@@ -63,15 +82,288 @@ class PlatformAllocator : public Allocator {
   std::string device_id_;
 };
 
```

Comment:
I feel like some of this code belongs somewhere other than ffi.cc

Reply:
Agreed. I’m currently working on a refactor and a way to configure passes based on the target. Was considering making a new dialect for pass configuration but after some experiments I’m currently thinking of sticking with protobuf.